bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/onvif/OnvifDiscovery.java

```diff
@@ -82,7 +88,15 @@ public OnvifDiscovery(IpCameraDiscoveryService ipCameraDiscoveryService) {
                     InetAddress inetAddress = enumIpAddr.nextElement();
                     if (!inetAddress.isLoopbackAddress() && inetAddress.getHostAddress().toString().length() < 18
                             && inetAddress.isSiteLocalAddress()) {
-                        results.add(networkInterface);
+                        if (inetAddress.getHostAddress().equals(primaryHostAddress)) {
+                            results.add(networkInterface);
+                            logger.info("Scanning network {} for any ONVIF cameras", primaryHostAddress);
```

Comment:
This could be reduced to debug level.

Reply:
Done, change has been made to debug level.